components/DiscussionsCard/DiscussionsCard.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react'
+import styles from './discussionsCard.module.scss'
+import {
+  ReplyIcon,
+  ThumbsupIcon,
+  ThumbsdownIcon,
+  TrashIcon,
+  PencilIcon
+} from '@primer/octicons-react'
+
+export type DiscussionsCardProps = {
+  discussionType: 'main' | string
+  username: string
+  userPic: string
+  timeStamp: string
+  content: string
+  likes: number
+  dislikes: number
+  expandClick: Function
+  likeClick: Function
+  dislikeClick: Function
+}
+
+const DiscussionsCard: React.FC<DiscussionsCardProps> = ({
+  discussionType,
+  username,
+  userPic,
+  timeStamp,
+  content,
+  likes,
+  dislikes,
+  expandClick,
+  likeClick,
+  dislikeClick
+}) => {
+  const mainType = discussionType === 'main'
+
+  return (
+    <div className={mainType ? styles.card_body_main : styles.card_body_sub}>
+      <div
+        className={
+          mainType ? styles.card_user_info_main : styles.card_user_info_sub
+        }
+      >
+        <div>
+          <img src={userPic} className={styles.user_pic} />
+          <span className={styles.username}>{username}</span>
+          <span> </span>
+          <span> - </span>
+          <span>{timeStamp}</span>
+        </div>
+        <div className={styles.expand_button} onClick={expandClick()}>
+          ...
```

Comment:
What does the expandClick function do in this case? I remember from the design sessions the idea behind was to use it to edit/delete the comment, but it looks like those two have their own button in this design, should we keep it?

Reply:
It depends. Do we want to show every response to the thread(including responses to responses) upon load, or do we want the user to click first before it expands?

Reply:
Interesting, i didn't think it would be used for collapsing comments.
I don't think hiding the content of the comment is a good idea, comments should all be visible and if the thread is too long, we can add a button to show more responses instead of making them collapsible.
Maybe we can make the comments who have a negative score collapsed by default.
Also do we have a design on the collapsed state?

Reply:
> Do we have a design on the collapsed state?

Only the discussion cards so far. Haven't implemented the full UI yet. I agree that we can load them all to a certain point. 
Should we have all responses feed to main parent post(the purple design), even if they were responding to a sub post? Or should we have it like reddit, where it continues branching out if you reply to a sub post?

Reply:
> Should we have all responses feed to main parent post(the purple design), even if they were responding to a sub post? Or should we have it like reddit, where it continues branching out if you reply to a sub post?

I think having something similar to reddit is the way to go, we just need to make sure that the design will not break once we have too many branches.